lib/quickdraw/cli.rb

```diff
@@ -24,6 +24,10 @@ def parse_options
 				exit
 			end
 
+			parser.on("-b", "--backtrace", "Backtrace past test into Quickdraw") do |it|
```

Comment:
I confess that I do not immediately grok this sentence.

Reply:
It means the backtrace should go past the tests and into the framework code.

Reply:
I was using `backtrace` as a verb. Maybe "Continue backtrace past test into Quickdraw" would be more obvious?